exasol/transformers_extension_wrapper.py

```diff
@@ -0,0 +1,239 @@
+    )
+
+    # Install the language container.
+    deployer.download_from_github_and_run(version, False)
+
+    # Save the activation SQL in the secret store.
+    activation_sql = deployer.generate_activation_command(
+        deployer.SLC_NAME, LanguageActivationLevel.Session
+    )
+    conf.save(ACTIVATION_KEY, activation_sql)
+
+
+def deploy_scripts(conf: Secrets,
+                   language_alias: str) -> None:
+    """
+    Deploys all the extension's scripts to the database.
+
+    Parameters:
+        conf:
+            The secret store. The store should contain the language activation
+            SQL command. This command should be created during the language
+            container deployment. The store should also have the DB schema.
+        language_alias:
+            The language alias of the extension's language container.
+    """
+
+    with open_pyexasol_connection(conf, compression=True) as conn:
+        # First need to activate the language container at the session level.
+        activation_sql = get_activation_sql(conf)
+        conn.execute(activation_sql)
+
+        scripts_deployer = ScriptsDeployer(language_alias, conf.SCHEMA, conn)
+        scripts_deployer.deploy_scripts()
+
+
+def initialize_te_extension(conf: Secrets,
+                            version: str = LATEST_KNOW_VERSION,
+                            language_alias: str = LANGUAGE_ALIAS,
+                            run_deploy_container: bool = True,
+                            run_deploy_scripts: bool = True,
+                            run_encapsulate_bfs_credentials: bool = True,
+                            run_encapsulate_hf_token: bool = True) -> None:
+    """
+    Performs all necessary operations to get the Transformers Extension
+    up and running. See the "Getting Started" and "Setup" sections of the
+    extension's User Guide for details.
+
+    Parameters:
+        conf:
+            The secret store. The store should contain all the required
+            parameters for accessing the database and bucket-fs.
+        version:
+            Transformers Extension version. If not specified the hardcoded
+            latest known version will be used.
+        language_alias:
+            The language alias of the extension's language container. Normally
+            this parameter would only be used for testing.
+        run_deploy_container:
+            If set to False will skip the language container deployment.
+        run_deploy_scripts:
+            If set to False will skip the deployment of the UDF scripts.
+        run_encapsulate_bfs_credentials:
+            If set to False will skip the creation of the database connection
+            object encapsulating the bucket-fs credentials.
+        run_encapsulate_hf_token:
+            If set to False will skip the creation of the database connection
+            object encapsulating the Huggingface token.
+    """
+
+    # Make the connection object names
```

Comment:
I usually prefer extraction to functions with good names instead of section comments